rules/aip0140/uri_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 		https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0140
+
+import (
+	"testing"
+
+	"github.com/googleapis/api-linter/rules/internal/testutils"
+)
+
+func TestURI(t *testing.T) {
+	for _, test := range []struct {
+		name     string
+		Field    string
+		problems testutils.Problems
+	}{
+		{"Valid", "uri", nil},
+		{"ValidPrefix", "foo_uri", nil},
```

Comment:
Nit: should we add a `ValidIntermixed` test that uses a field name like `curl_uri` with "url" inside a word to ensure it _doesn't_ emit an error? I understand that the implementation splits on `_` and does a `Contains` check on the slice contents, but implementations change, and the behavior shouldn't necessarily.

```suggestion
		{"ValidPrefix", "foo_uri", nil},
		{"ValidIntermixed", "curl_uri", nil},
```

Reply:
Good idea.

Reply:
Done. Still not merging until @jskeet approves the AIP change (probably Monday).